ci-operator/config/openshift/cluster-kube-apiserver-operator/openshift-cluster-kube-apiserver-operator-main.yaml

```diff
@@ -451,6 +451,15 @@ tests:
   steps:
     cluster_profile: azure-2
     workflow: openshift-e2e-cert-rotation-short-azure
+- as: e2e-gcp-operator-ote
+  optional: true
+  steps:
+    cluster_profile: gcp-3
+    env:
+      TEST_SUITE: openshift/cluster-kube-apiserver-operator/operator/all
```

Comment:
this should be `openshift/cluster-kube-apiserver-operator/operator/serial`, no ?